myuw/dao/term.py

```diff
@@ -391,6 +403,10 @@ def add_term_data_to_context(request, context):
 
     context['first_day'] = cur_term.first_day_quarter
     context['last_day'] = cur_term.last_day_instruction
+    context["first_day_quarter"] = cur_term.first_day_quarter
+    context["last_day_instruction"] = cur_term.last_day_instruction
```

Comment:
Academic calendar specifies final week as Saturday to Friday:  http://www.washington.edu/students/reg/1617cal.html#Q2.  Our final week starts one day later.